### ISSUE TITLE
Web2Exe broke printing

### ISSUE THREAD

**Issue by Forecaster** [jyapayne/Web2Executable#198]
For this one project I'm making an application that handles data, and one window generates a summary of this data that you're supposed to be able to print.

When I was working on this and added the print button it was working fine, I got the chrome print dialog with the preview as expected.

However after packaging the application and running it I tested the printing function it was broken. The window now said something along the lines of "NW.js is missing something to support previews". The message was in Swedish because my work computer is installed in that language, so I don't know what the proper message would be.

I was confounded. I had no idea what I'd done to break it. And I thought it was just the packaging so I tried in my dev environment and it wasn't working there anymore either.

Just a few minutes ago I was going into the package.json to bump the version up to 1.0.0 for the first release and found that Web2Exe had changed it. Presumably I forgot to open the project in readonly mode at one point.

I restored it from my IDE's history to how it was before then, and decided to check printing again which now worked as before.

So something Web2Exe did to my package.json broke the printing feature. I've yet to determine what. But I will comment with more information should I get any.

**Comment by Forecaster** [comment]
It might be this bit. I don't know what webkit plugin is for.
```
"webkit": {
    "plugin": false
  }
```

**Comment by Blatman** [comment]
Yes indeed - nwjs uses the PDF plugin for print previews so the webkit plugin needs to be true.
